test/azure/scanner.spec.ts

```diff
@@ -1,24 +1,32 @@
 import {assert} from 'chai';
 import {Scanner} from '../../src/scanner';
-import {subscriptionId} from '.';
+import {resourceGroup, subscriptionId} from '.';
+import {ResourceGraphTarget, RuleType} from '../../src/rules';
 
 describe('Scanner', function () {
   this.slow(5000);
   this.timeout(8000);
   it('can load and execute resource graph rules from a JSON file', async () => {
+    const rgTarget: ResourceGraphTarget = {
+      type: RuleType.ResourceGraph,
+      subscriptionId,
+    };
     const scanner = new Scanner();
     await scanner.loadRulesFromFile('../test/rules.json');
-    const rgResults = await scanner.scan('resourceGraph', subscriptionId);
-    assert.equal(rgResults.length, 1);
+    const rgResults = await scanner.scan(rgTarget);
+    const re = new RegExp(`/resourceGroups/${resourceGroup}`);
+    assert.isAtLeast(rgResults.length, 2);
```

Comment:
isAtLeast - we should know the exact answers we're looking for in our test cases here

Reply:
I agree.